ROADMAP.md

```diff
@@ -0,0 +1,22 @@
+# Our Roadmap
+
+Hey, there. Thanks so much for the interest in openSNP and what we're trying to accomplish over the next weeks and months.
+
+We have some ongoing efforts which need to be done as well as some bug fixes and new features we'd love to implement. If you think we should focus on something else: Please just comment, add it or talk to us, whatever you feel like doing!
+
+
+## Ongoing Efforts
```

Comment:
It might be as simple as adding a bit of text under that section: "These issues are up next to be worked on! This is a good place to jump in if you want to help with some of our heavier tasks"